lib/galaxy/tools/deps/conda_util.py

```diff
@@ -0,0 +1,305 @@
+import functools
+import hashlib
+import os.path
+import re
+import shutil
+from sys import platform as _platform
+import tempfile
+
+import six
+import yaml
+
+from ..deps import commands
+
+# Not sure there are security concerns, lets just fail fast if we are going
+# break shell commands we are building.
+SHELL_UNSAFE_PATTERN = re.compile(r"[\s\"']")
+
+IS_OS_X = _platform == "darwin"
+
+# BSD 3-clause
+CONDA_LICENSE = "http://docs.continuum.io/anaconda/eula"
+
+
+def conda_link():
+    if IS_OS_X:
+        url = "https://repo.continuum.io/miniconda/Miniconda-latest-MacOSX-x86_64.sh"
+    else:
+        url = "https://repo.continuum.io/miniconda/Miniconda-latest-Linux-x86_64.sh"
```

Comment:
Galaxy doesn't ship with its dependencies like wheels so my instinct is probably not to do this. Maybe if someday there is an all-in-one deb sort of install it could make a lot of sense.